#
# adjusting expected output, looks like ftp client library is not able to deal
# with ABOR command reply properly, change  has been submitted to upstream:
#	https://github.com/proftpd/proftpd/pull/1512
#

--- a/tests/t/lib/ProFTPD/Tests/Config/DisplayFileTransfer.pm
+++ b/tests/t/lib/ProFTPD/Tests/Config/DisplayFileTransfer.pm
@@ -136,7 +136,7 @@ sub displayfilexfer_abs_path {
       my ($resp_code, $resp_msg);
 
       $resp_code = $client->response_code();
-      $resp_msg = $client->response_msg(1);
+      $resp_msg = $client->response_msg(0);
 
       my $expected;
 
@@ -144,7 +144,7 @@ sub displayfilexfer_abs_path {
       $self->assert($expected == $resp_code,
         test_msg("Expected response code $expected, got $resp_code"));
 
-      $expected = "Hello user!";
+      $expected = "Abort successful";
       $self->assert($expected eq $resp_msg,
         test_msg("Expected response message '$expected', got '$resp_msg'"));
     };
@@ -263,7 +263,7 @@ sub displayfilexfer_rel_path {
       my ($resp_code, $resp_msg);
 
       $resp_code = $client->response_code();
-      $resp_msg = $client->response_msg(1);
+      $resp_msg = $client->response_msg(0);
 
       my $expected;
 
@@ -271,7 +271,7 @@ sub displayfilexfer_rel_path {
       $self->assert($expected == $resp_code,
         test_msg("Expected response code $expected, got $resp_code"));
 
-      $expected = "Hello user!";
+      $expected = "Abort successful";
       $self->assert($expected eq $resp_msg,
         test_msg("Expected response message '$expected', got '$resp_msg'"));
     };
@@ -539,7 +539,8 @@ sub displayfilexfer_multiline {
       my ($resp_code, $resp_msg);
 
       $resp_code = $client->response_code();
-      $resp_msg = $client->response_msg(3);
+      $resp_msg = $client->response_msg();
+
 
       my $expected;
 
@@ -547,7 +548,7 @@ sub displayfilexfer_multiline {
       $self->assert($expected == $resp_code,
         test_msg("Expected response code $expected, got $resp_code"));
 
-      $expected = " Hello user!";
+      $expected = "Abort successful";
       $self->assert($expected eq $resp_msg,
         test_msg("Expected response message '$expected', got '$resp_msg'"));
     };
